### PR TITLE
Moved archive cookiecutter templates

### PR DESCRIPTION
**Pull request by eudesbarbosa**
addresses #96 

**Changes**
* Moved `archive` cookiecutter directory - isolated ISA templates.
* Moved config files away from template directory.

### CODE REVIEW

**Comment by eudesbarbosa** [comment]
@messersc, do you mind checking if I missed something?
...and also to review when we agree it is done?

**Comment by eudesbarbosa** [comment]
Agreed. Added the suggestion in today's commit.

File: cubi_tk/archive/common.py [hunk 1]
```diff
@@ -117,6 +118,19 @@ def traverse_project_files(directory, followlinks=True):
             yield get_file_attributes(os.path.join(path, filename), root)
 
 
+def load_variables(template_dir):
```
Comment (eudesbarbosa) [inline cubi_tk/archive/common.py:121]: I simplified this function from the `isa_tpl` version. Just to check, is the `extra` argument used for anything in the `archive`?

Comment (ericblanc20) [reply cubi_tk/archive/common.py:121]: Not in use, I don't see any situation which would require programmatically updating the configuration.

**Review by ericblanc20 — APPROVED** [review]

**Pull request merged**